unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UserContextMenu/Scripts/UserContextMenu.cs

```diff
@@ -130,9 +131,20 @@ public void Show(string userId, MenuConfigFlags configFlags)
     /// <param name="userName">User name</param>
     public void ShowByUserName(string userName)
     {
-        var userProfile = UserProfileController.userProfilesCatalog.GetValues().FirstOrDefault(p => p.userName.ToLower() == userName.ToLower());
+        var userProfile = UserProfileController.userProfilesCatalog
+                                               .GetValues()
+                                               .FirstOrDefault(p => p.userName.Equals(userName, StringComparison.OrdinalIgnoreCase));
```

Comment:
Do you think we can avoid Linq here?

Reply:
The user profile catalog feature needs an entire refactor. Not the purpose of this pr, i just added new lines.